Basic/Pod/QuickStart.pod

```diff
@@ -284,18 +284,18 @@ of, for example, the PGPLOT FORTRAN library.
 
 Following the usual convention coordinate (0,0) is displayed
 at the bottom left when displaying an image. It appears at the
-top left when using "C<print $a>" etc.
+top left when using "C<print $x>" etc.
 
 Simple sectioning uses a syntax extension to Perl,
 L<PDL::NiceSlice|PDL::NiceSlice>, that allows you to specify subranges
 via a null-method modifier to a PDL:
 
-  $b = $a->($x1:$x2,$y1:$y2,($z1)); # Take subsection
+  $y = $x->($x1:$x2,$y1:$y2,($z1)); # Take subsection
```

Comment:
Suggest not using `$x` and `$y` here (and in the paragraph below) since docs are using `$x1`, `$y1` etc to describe slice limits.